.github/auto-comment.yml

```diff
@@ -0,0 +1,71 @@
+pullRequestOpened: |
```

Comment:
It would be best if rolling back this removal didn't show in the diff.

Reply:
@oesteban TBC are you requesting a rebase so that it's not in the history at all? The change is not visible in the PR "files changed" tab unless you look at specific commits.

Reply:
Oh right, yes, then it's okay I don't think editing the history is worth the pain.